set.mm

```diff
@@ -732651,6 +732663,410 @@ The modulo (remainder) operation - extension
+        ( vs vr vb va wf cv cfv wceq wi wa wral wf1 uniimaelsetpreimafv wfn ffn
+        cima cuni fmptd wcel fundcmpsurinjlem5 ex anim12d syl imp wel wb eqeq12
+        wrex ancoms adantl ad3antrrr simpr anim1i elsetpreimafveq adantr sylbid
+        simpllr syl3anc exp32 rexlimdva com23 impd ralrimivva dff13 sylanbrc
+        mpd ) CDFOZEDGOKPZGQZLPZGQZRZVRVTRZSZLEUAKEUAEDGUBVQHEFHPZUFUGDGABCDEWE
+        FIUCJUHVQWDKLEEVQVREUIZVTEUIZTZTZVSMPZFQZRZMVRURZWANPZFQZRZNVTURZTZWDVQ
+        WHWRVQFCUDZWHWRSCDFUEZWSWFWMWGWQWSWFWMAMBCEFGVRHIJUJUKWSWGWQANBCEFGVTHI
+        JUJUKULUMUNWIWMWQWDWIWLWQWDSMVRWIMKUOZTZWQWLWDXBWPWLWDSNVTXBNLUOZTZWPWL
+        WDXDWPWLTZTWBWKWORZWCXEWBXFUPZXDWLWPXGVSWKWAWOUQUSUTXDXFWCSZXEXDWSWHXAX
+        CTXHVQWSWHXAXCWTVAVQWHXAXCVGXBXAXCWIXAVBVCABCEVTVRFWJWNIVDVHVEVFVIVJVKV
+        JVLVPVMKLEDGVNVO $.
+    $}
+
+    $d A g h p y $.  $d B g h $.  $d B p x y z $.  $d F g h p y $.
+    $d P g h p x y $.  $d V x y $.
+    $( Every function ` F : A --> B ` can be decomposed into a surjective
+       function onto ` P ` and an injective function from ` P ` .  (Contributed
+       by AV, 12-Mar-2024.) $)
+    fundcmpsurinjpreimafv $p |- ( ( F : A --> B /\ A e. V ) -> E. g E. h
+                  ( g : A -onto-> P /\ h : P -1-1-> B /\ F = ( h o. g ) ) ) $=
+      ( vy vp wcel wa cv cima cmpt cvv wceq adantr wf ccnv cfv csn cuni wfo wf1
+      ccom w3a wex mptexg adantl setpreimafvex mptexd wfn ffn fundcmpsurinjlem2
+      wb fveq2 sneqd imaeq2d cbvmptv foeq1 ax-mp sylibr sylan fundcmpsurinjlem6
+      eqid uniimaprimaeqfv mpteq2dva simpr preimafvelsetpreimafv syl3anc imaeq2
+      eqidd unieqd fmptco dffn5 sylib 3eqtr4rd coeq12d eqeq2d 3anbi123d spc2egv
+      f1eq1 simpl imp syl23anc ) CDHUAZCIMZNZKCHUBZKOZHUCZUDZPZQZRMZLEHLOZPZUEZ
+      QZRMZCEWQUFZEDXBUGZHXBWQUHZSZCEFOZUFZEDGOZUGZHXJXHUHZSZUIZGUJFUJZWJWRWIKC
+      WPIUKULWKLEXARWJERMWIABCEHIJUMULUNWIHCUOZWJXDCDHUPZXPWJNCEACWLAOZHUCZUDZP
+      ZQZUFZXDABCEHYBIJYBVHUQWQYBSXDYCURKACWPYAWMXRSZWOXTWLYDWNXSWMXRHUSUTVAVBC
+      EWQYBVCVDVEVFWIXEWJABCDEHXBLJXBVHVGTWKKCHWPPZUEZQKCWNQZXFHWKKCYFWNWKXPWMC
+      MZYFWNSWIXPWJXQTZCHWMVIVFVJWKKLCEWPXAYFWQXBWKYHNXPWJYHWPEMWKXPYHYITWKWJYH
+      WIWJVKTWKYHVKABCEHIWMJVLVMWKWQVOWKXBVOWSWPSWTYEWSWPHVNVPVQWIHYGSZWJWIXPYJ
+      XQKCHVRVSTVTWRXCNXDXEXGUIZXOXNYKFGWQXBRRXHWQSZXJXBSZNZXIXDXKXEXMXGYLXIXDU
+      RYMCEXHWQVCTYMXKXEURYLEDXJXBWEULYNXLXFHYNXJXBXHWQYLYMVKYLYMWFWAWBWCWDWGWH
+      $.
+  $}
+
+  ${
+    $d A g h p x y z $.  $d B g h p x y z $.  $d F g h p x y z $.  $d V x y $.
+    $( Every function ` F : A --> B ` can be decomposed into a surjective and
+       an injective function.  (Contributed by AV, 13-Mar-2024.) $)
+    fundcmpsurinj $p |- ( ( F : A --> B /\ A e. V ) -> E. g E. h E. p
+                   ( g : A -onto-> p /\ h : p -1-1-> B /\ F = ( h o. g ) ) ) $=
+      ( vz vx vy wcel cv wfo wf1 wceq w3a wex cfv csn cima wf wa ccom ccnv wrex
+      cab cvv abrexexg adantl fveq2 sneqd imaeq2d cbvrexv fundcmpsurinjpreimafv
```

Comment:
Same here again, there is no tradeoff in using [cbvrexvw](https://us.metamath.org/mpeuni/cbvrexvw.html) instead of [cbvrexv](https://us.metamath.org/mpeuni/cbvrexv.html).
<pre>
      ( vz vx vy wcel cv wfo wf1 wceq w3a wex cfv csn cima wf wa ccom ccnv wrex
      cab cvv abrexexg adantl fveq2 sneqd eqeq2d cbvrexvw fundcmpsurinjpreimafv
      imaeq2d abbii foeq3 f1eq2 3anbi12d 2exbidv elabd exrot3 sylib ) ABEUAZAFK
      ZUBZAGLZCLZMZVGBDLZNZEVJVHUCOZPZDQCQZGQVMGQDQCQVFVNAHLZEUDZILZERZSZTZOZIA
      UEZHUFZVHMZWCBVJNZVLPZDQCQGWCVEWCUGKVDIHAVTFUHUIJHABWCCDEFWBVOVPJLZERZSZT
      ZOZJAUEHWAWKIJAVQWGOZVTWJVOWLVSWIVPWLVRWHVQWGEUJUKUOULUMUPUNVGWCOZVMWFCDW
      MVIWDVKWEVLVGWCAVHUQVGWCBVJURUSUTVAVMGCDVBVC $.
</pre>

As a rule of thumb, you'll basically never need the stronger versions.

Reply:
Done